### PR TITLE
Create Policy classes and storage class for policy

### LINKED ISSUE #101: Create policy class

### LINKED ISSUE #106: Create Storage for PolicyList

### PR DESCRIPTION
**Pull request by ijavierja**
close #106 
close #101

### CODE REVIEW

File: src/main/java/seedu/address/model/policy/Policy.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ */
+public class Policy {
+
+    private final PolicyName policyName;
+    private final PolicyDescription description;
+
+    /**
+     *
+     * @param policyName
+     * @param description
```
Comment (WeiJie96) [inline src/main/java/seedu/address/model/policy/Policy.java:18]: Perhaps we should add in javadoc? Same for the rest of this class.

File: src/main/java/seedu/address/model/person/exceptions/DuplicatePolicyException.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.person.exceptions;
```
Comment (WeiJie96) [inline src/main/java/seedu/address/model/person/exceptions/DuplicatePolicyException.java:1]: Perhaps this should be in policy.exceptions?

File: src/main/java/seedu/address/model/policy/Policy.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    @Override
+    public int hashCode() {
+        return policyName.hashCode();
+    }
+
+    /**
+     * Returns true if otherPolicy has the same name and same description.
+     * This method differs from equals method as isSamePolicy takes into account
+     * both the name and description.
```
Comment (WeiJie96) [inline src/main/java/seedu/address/model/policy/Policy.java:56]: Should this be only takes into account of name?

File: src/main/java/seedu/address/storage/JsonAdaptedPolicyList.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
+package seedu.address.storage;
```
Comment (WeiJie96) [inline src/main/java/seedu/address/storage/JsonAdaptedPolicyList.java:1]: Perhaps we should rename this file/class to JsonSerializablePolicyList?

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Good in general, just some minor comments

File: src/main/java/seedu/address/storage/JsonAdaptedPolicyList.java (hunk 4 above)
Comment (ijavierja) [reply src/main/java/seedu/address/storage/JsonAdaptedPolicyList.java:1]: Ok sounds good too. Can change it.

File: src/main/java/seedu/address/model/policy/Policy.java (hunk 3 above)
Comment (ijavierja) [reply src/main/java/seedu/address/model/policy/Policy.java:56]: Right. Thanks for pointing it out

File: src/main/java/seedu/address/model/person/exceptions/DuplicatePolicyException.java (hunk 2 above)
Comment (ijavierja) [reply src/main/java/seedu/address/model/person/exceptions/DuplicatePolicyException.java:1]: Yes. Will change this.

File: src/main/java/seedu/address/model/policy/Policy.java (hunk 1 above)
Comment (ijavierja) [reply src/main/java/seedu/address/model/policy/Policy.java:18]: Changed this. thanks for pointing it out

**Review by WeiJie96 — APPROVED** [review]
LGTM

**Pull request merged**